irc/proto/src/command.rs

```diff
@@ -229,6 +243,10 @@ impl Command {
             Command::USERHOST(params) => params,
             Command::WALLOPS(a) => vec![a],
             Command::BATCH(a, rest) => std::iter::once(a).chain(rest).collect(),
+            Command::CNOTICE(a, b, c) => vec![a, b, format!(":{}", c)],
+            Command::CPRIVMSG(a, b, c) => vec![a, b, format!(":{}", c)],
```

Comment:
Don't add `:` here, it's handled during `proto::format::parameters`

Reply:
And *now* I understand what `:` means.  Thank you!